web/src/model/project-actions.js

```diff
@@ -228,27 +295,30 @@ export const installProject = (catalog, name, onSuccess, onFailure) => dispatch
       if (onFailure) {
         onFailure(failureResult);
       }
-    });
+    },
+  );
 };
 
-export const orderResultsByProjectName = (a, b) => a.name < b.name ? -1 : a.name > b.name ? 1 : 0;
+export const orderResultsByProjectName = (a, b) => a.name.localeCompare(b.name);
```

Comment:
Fix an error due to nested ternary-if statements. I replaced the logic with `String.prototype.localeCompare` which has the same behavior.